repository/bolt.go

```diff
@@ -0,0 +1,89 @@
+package repository
+
+import (
+	"encoding/binary"
+	"errors"
+	"fmt"
+
+	"github.com/boltdb/bolt"
+)
+
+const (
+	riotBucket       = "riot"
+	riotVerifyBucket = "verify"
+	riotQuotesBucket = "quotes"
+	generalBucket    = "general"
```

Comment:
i should probably declare the key names within each bucket here.. ( i think it's jsut 'commands')